public/tenants/planet/config.ts

```diff
@@ -36,13 +36,50 @@ const config = {
         title: 'aboutUs',
         onclick: 'https://a.plant-for-the-planet.org/',
         visible: true,
+        subMenu: [
+          {
+            title: 'Overview',
```

Comment:
This solution does not consider translating resources. Please use the `title` as a key for a translation resource, e.g. `overview` -> `Overview` as at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/ef8e93859407c7d9a2cf91a26c6120e3b1c001da/src/features/common/Layout/Navbar/index.tsx#L162